### PR TITLE
Adjust Readme

### PR DESCRIPTION
**Pull request by chjaeggi**
+ Add installation/implementation advice/instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
 # embedded-common
 Embedded Driver common files
+
+## Clone this repository
+If you really need this repository you can check it out with the following command:
```
Comment (abrauchli) [inline README.md]: Cloning instruction is not needed here, I'd add the following note instead (also remove the subsection header "Clone this repository").

> This repository contains the Sensirion i2c abstraction layer and commonly shared code. It is a dependency of the Sensirion embedded drivers and it should not be necessary to clone it independently.

Then follow up with the last paragraph "We recommend to download ..."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+```
+We recommend to download the repository recursively within the [SGP](https://github.com/Sensirion/embedded-sgp) and [SHT](https://github.com/Sensirion/embedded-sht) projects.
+
+## Repistory content
+* *.c files (CRC calculation function)
+* *.h files (type definitions, prototype declarations for i2c communication,...)
+* hw_i2c folder (stub for implementation of the hardware i2c communication, including sample code for some platforms)
+* sw_i2c folder (stub for implementation of the software i2c communication (bit-banging GPIOs), including sample code for some platforms)
+
+## Where to start?
```
Comment (abrauchli) [inline README.md]: I'd kill the whole subsection..

File: README.md [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+* u64 = unsigned 64bit integer
+* s32 = signed 32bit integer
+* u32 = unsigned 32bit integer
+* s16 = signed 16bit integer
+* u16 = unsigned 16bit integer
+* s8 = signed 8bit integer
+* u8 = unsigned 8bit integer
+
+Make sure you specify the whole range (64bit sized integers are used on a 32bit system for some sensor data).
+adi
```
Comment (abrauchli) [inline README.md]: ?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,39 @@
...
+If you really need this repository you can check it out with the following command:
+```
+git clone https://github.com/Sensirion/embedded-common.git
+```
+We recommend to download the repository recursively within the [SGP](https://github.com/Sensirion/embedded-sgp) and [SHT](https://github.com/Sensirion/embedded-sht) projects.
+
+## Repistory content
+* *.c files (CRC calculation function)
+* *.h files (type definitions, prototype declarations for i2c communication,...)
+* hw_i2c folder (stub for implementation of the hardware i2c communication, including sample code for some platforms)
```
Comment (abrauchli) [inline README.md]: Add backticks around hw_i2c and sw_i2c for visual distinction -> `hw_i2c`

File: README.md [hunk 5]
```diff
@@ -1,2 +1,39 @@
...
+* u32 = unsigned 32bit integer
+* s16 = signed 16bit integer
+* u16 = unsigned 16bit integer
+* s8 = signed 8bit integer
+* u8 = unsigned 8bit integer
+
+Make sure you specify the whole range (64bit sized integers are used on a 32bit system for some sensor data).
+adi
+### hw_i2c_implementation.c
+In this file you implement the hardware i2c communication on your system provided there is one available. You will find four function stubs that you have to implement out. There is an init, a read, a write and a delay function. For further information please check out some of the sample implementations in the hw_i2c/sample-implementations folder in this repository.
```
Comment (abrauchli) [inline README.md]: Add backticks around path for visual distinction -> `hw_i2c/sample-implementations` (and in the next subsection for sw_i2c)

Comment (chjaeggi) [reply README.md]: I'll do that thanks

File: README.md (hunk 3 above)
Comment (chjaeggi) [reply README.md]: adi will be removed

File: README.md (hunk 2 above)
Comment (chjaeggi) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (chjaeggi) [reply README.md]: Done

**Comment by chjaeggi** [comment]
So we should do that to any readme then (in the other repos as well?)

**Comment by abrauchli** [comment]
Yes, that'd be great. I can also do it - vim knows tricks ;)

**Pull request merged**